include/MainWindow.h

```diff
@@ -81,9 +82,25 @@ class MainWindow : public QMainWindow
 	///
 	bool mayChangeProject(bool stopPlayback);
 
-	void autoSaveTimerStart()
+	// Auto save timer intervals
+	static const int DEFAULT_AUTO_SAVE_INTERVAL = 2;
+	int m_autoSaveInterval;
+	const static int m_autoSaveShortTime = 10 * 1000; // 10s short loop
+
+	void autoSaveTimerReset( int msec = ConfigManager::inst()->
```

Comment:
The times I had problems with was when the user has auto save on but no time setting. People who upgrade to 1.2 . I'm a bit unsure how the whole init part and it gets a zero if there is no previous setting.